app/helpers/profile_helpers.py

```diff
@@ -177,7 +177,7 @@ def _fill(coordinates, nb_points, is_smart=False):
                         nb_points_placed += 1
                         segment_length_covered += segment_resolution
                         new_point = segment.interpolate(nb_points_placed * segment_resolution)
-                        result.append([new_point.x, new_point.y])
+                        result.append([new_point.x, new_point.y])  # pylint: disable=no-member
```

Comment:
Those pylint pragma could be removed and replaced by a pylintrc config; `ignored-classes=optparse.Values,thread._local,_thread._local,BaseGeometry`